IdentityCore/src/parameters/MSIDInteractiveRequestParameters.h

```diff
@@ -34,7 +34,7 @@
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->